test/gtest/ucs/test_log.cc

```diff
@@ -40,13 +40,17 @@ class log_test : public ucs::test {
         modify_config("LOG_FILE", ucs_log_spec);
         modify_config("LOG_LEVEL", "info");
         ucs_log_init();
+
+        logfile_fp = fopen(logfile, "r");
```

Comment:
can you use `ucs_log_file` instead?

Reply:
looks like no. plz disregard

Reply:
this is not exposed through `ucs/debug/log.h` header file, so, we don't have access to it